modules/pytket-projectq/docs/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 ~~~~~~~~~
 
+0.12.0 (XXXX 2021)
+------------------
+
+* Updated projectq version requirement to 0.7.
```

Comment:
Why is this here?

Reply:
Ah I see, it is fixing up the previous PR.

Reply:
This wasn't added to a previous PR (my fault). Or should I put this somewhere else?